src/sql/statement.rs

```diff
@@ -627,28 +627,54 @@ fn handle_show_create_sink(
     ])]))
 }
 
-fn handle_create_sink(scx: &StatementContext, stmt: Statement) -> Result<Plan, failure::Error> {
-    let create_sql = normalize::create_statement(scx, stmt.clone())?;
-    let (name, from, connector, format, if_not_exists) = match stmt {
-        Statement::CreateSink {
-            name,
-            from,
-            connector,
-            format,
-            if_not_exists,
-        } => (name, from, connector, format, if_not_exists),
-        _ => unreachable!(),
-    };
+fn handle_create_avro_ocf_sink(
```

Comment:
this refactor could be a little nicer -- have the helpers generate a Sink and let handle create sink make the plan?